node/src/BaseClient.ts

```diff
@@ -297,6 +297,22 @@ export type DecoderOption = {
     decoder?: Decoder;
 };
 
+export function convertFieldsAndValuesForHset(
```

Comment:
I guess we can't export such function, TBD

Reply:
As discussed, we are keeping it this way.